data/presets/barrier/hampshire_gate.json

```diff
@@ -0,0 +1,13 @@
+{
+    "icon": "temaki-gate",
+    "fields": [
+        "access"
+    ],
+    "geometry": [
+        "vertex"
+    ],
+    "tags": {
+        "barrier": "hampshire_gate"
+    },
+    "name": "Hampshire Gate"
```

Comment:
Should we add `New Zealand Gate` and `Wire Gate` as aliases or terms (from https://en.wikipedia.org/wiki/Hampshire_gate)?